### PR TITLE
PP-8902 add final content for KYC implementation

### PR DESCRIPTION
**Pull request by SandorArpa**
## WHAT
- update pages with content finalised in [KYC signed-off content](https://docs.google.com/document/d/1-_ohNR4DjT26alRaIAdu2cR5Rqq9TEbhGYJEkoFlQkA)
- update Cypress tests

### CODE REVIEW

**Review by kbottla — CHANGES_REQUESTED** [review]

File: app/views/request-to-go-live/index.njk [hunk 1]
```diff
@@ -58,33 +59,26 @@
         <p class="govuk-body-l intro">Complete these steps to request a live account.</p>
       {% endif %}
       <div class="request-to-go-live-list">
-        <div id="request-to-go-live-step-organisation-name">
-          <h2 class="govuk-heading-m">
-            Add your organisation’s name and address
+        <ol class="govuk-list govuk-list--number">
```
[the remark this answers is not in the document]
Comment (SandorArpa) [reply app/views/request-to-go-live/index.njk:62]: Made grid column full so everything appears on one line.

File: app/views/request-to-go-live/organisation-address.njk [hunk 2]
```diff
@@ -110,15 +110,18 @@
 
     {% call govukFieldset({
     legend: {
-    text: "Organisation address",
-    classes: "govuk-fieldset__legend--m",
-    isPageHeading: false
+        text: "Enter your organisation’s contact details",
```
[the remark this answers is not in the document]
Comment (SandorArpa) [reply app/views/request-to-go-live/organisation-address.njk]: Reverted to Stephen's changes that already includes the wanted changes.

File: app/views/request-to-go-live/organisation-address.njk [hunk 3]
```diff
@@ -110,15 +110,18 @@
...
+        classes: "govuk-fieldset__legend--l govuk-!-margin-bottom-4",
```
[the remark this answers is not in the document]
Comment (SandorArpa) [reply app/views/request-to-go-live/organisation-address.njk]: Copied fixes from [#3064](https://github.com/alphagov/pay-selfservice/pull/3064)

File: app/views/request-to-go-live/choose-how-to-process-payments.njk [hunk 4]
```diff
@@ -80,9 +80,6 @@
             },
             conditional: {
               html: otherPSP
-            },
```
[the remark this answers is not in the document]
Comment (SandorArpa) [reply app/views/request-to-go-live/choose-how-to-process-payments.njk]: I re-added hint, this option is might be in use (we should remove it when we all services has been switched)

File: app/views/dashboard/_additional_kyc_info_banner.njk [hunk 5]
```diff
@@ -5,14 +5,19 @@
...
+      <p>
+          You have until {{ stripeKycAdditionalDataDueDate }} to add these details. If you do not add them, your service will stop taking payments and Stripe will stop paying out to your bank account.
       </p>
-
       <p class="govuk-body">
         Continue to
         <a id="add-additional-kyc-details" class="govuk-notification-banner__link"
            href="{{ formatAccountPathsFor(routes.account.yourPsp.index, currentGatewayAccount.external_id, activeCredential.external_id) }}">
-          add KYC details </a>.
+          Add new KYC Information </a>.
```
[the remark this answers is not in the document]
Comment (SandorArpa) [reply app/views/dashboard/_additional_kyc_info_banner.njk]: Removed white space

File: app/views/request-to-go-live/index.njk (hunk 1 above)
[the remark this answers is not in the document]
Comment (kbottla) [reply app/views/request-to-go-live/index.njk:62]: Do the tasks need to be in bold ?

[the remark this answers is not in the document]
Comment (SandorArpa) [reply app/views/request-to-go-live/index.njk:62]: Added to the css `font-weight: bold`, the class is only used here

**Review by kbottla — APPROVED** [review]
LGTM!

**Pull request merged**